integ-test/build.gradle

```diff
@@ -188,6 +189,8 @@ task compileJdbc(type: Exec) {
 String bwcVersion = "1.13.2.0";
 String baseName = "sqlBwcCluster"
 String bwcFilePath = "src/test/resources/bwc/"
+String bwcOpenDistroPlugin = "opendistro-sql-1.13.2.0.zip"
+String bwcRemoteFile = 'https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-sql/opendistro-sql-1.13.2.0.zip'
```

Comment:
How do we maintain this zip file in cloudfront? Like if this package has merged a new bug fix commit, do we need to build the zip file locally then upload it to cloudfront? Then although it's not a checked-in dependency anymore, it's still not a dynamic dependency?

Reply:
opendistro-sql-1.13.2.0.zip is the released latest opendistro SQL plugin. we didn't maintain opendistro-sql-1.13.2.0.zip

Reply:
if opendistro SQL plugin release a new version, do we need to replace this zip file on cloudfront with the one build with latest version?

Reply:
What's the official URL for this .zip? I see in https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/ that we reference the cloudfront URL. If that's how it should be, so be it, but I would use something like opendistrowhateverthatwas.org if that exists.